recipe/meta.yaml

```diff
@@ -62,6 +67,7 @@ requirements:
     - llvm-openmp                                         # [osx] bounds through run_exports
     - _openmp_mutex                                       # [linux]
     - __osx >={{ MACOSX_DEPLOYMENT_TARGET|default("10.12") }}  # [osx and x86_64]
+    - _x86_64-microarch-level >=3                         # [x86_64_opt == "v3"]
```

Comment:
this will then install the v3 variant on x86_64_v4 processors. the binaries will work fine, but it could lead to confusion for the future? could s/v3/v34 in the recipe and the build string to be clearer, for example

Reply:
The build number magic is taking care of that. In a way the no opt variant could be v01234.

Reply:
well, the v3 variant will install preferentially on v4 cpus. So the opt=none variant is more like v012?

Reply:
opt=none would be installable anywhere really